### DISCUSSION TITLE
Badges to display REST API Pod & Field status

### DISCUSSION THREAD

**Discussion by lougreenwood** [pods-framework/pods#6753] [Ideas]
Editing or auditing the REST API status of fields can be very tedious in WP Admin, this is especially true if REST fields are enabled on a per-field basis.

Instead, it would be useful if, when editing an individual Pod:
* If the post has REST enabled, display some badge next to the Pod name
    * If the post has REST 'Show all fields' enabled, display a `REST R` badge for all fields
    * If the post has REST 'Allow all fields to be updated' enabled, display a `REST W` badge for all fields
    * If a Pod has REST visibility manually managed (aka not 'Show all/Update all')
        * Display the same `REST RW` badges for fields

This would be a tremendous help for checking which fields have REST enabled. Currently on large sites either this is a very long process in WP Admin, or must be checked with actual REST responses (even more tedious when checking if writing is enabled).

It could look something like this:
<img width="1148" alt="screen shot 2017-10-18 at 10 00 00" src="https://user-images.githubusercontent.com/12051736/31710056-cc3044c2-b3eb-11e7-9bd1-89bf9c78ee3f.png">

**Comment by quasel** [comment]
a good idea to improve the interface!

**Comment by sc0ttkclark** [comment]
I like this, it'll require some React work but we'll see when we can fit this in

**Comment by JoryHogeveen** [comment]
Nice!
